### ISSUE TITLE
Buttons and keyboard shortcuts do not indicate than an action has been performed

### ISSUE THREAD

**Issue by scott-redrup-ou** [tinymce/tinymce#10107]
## 📝 Provide a description of the new feature or improvement
When toolbar buttons or keyboard shortcuts are used to perform actions (e.g., bold, italicize, insert link), screen readers do not provide feedback indicating that the action has been successfully executed. This lack of feedback may confuse users who rely on screen readers, as they cannot confirm if the formatting or insertion action was applied. For visually impaired users, this hinders the editing process, as they may not know if their input was effected. 

The recommendation is to implement accessible status messages using ARIA live regions for screen reader users that announce the result of the action (e.g., 'Bolded' or 'Link inserted') when a toolbar button or hotkey action is performed. Ensure these messages are concise and do not interrupt the editing flow. 

### 🫶 What is the motivation?
This feature is particularly useful when formatting content and for compliance with WCAG: 4.1.3 Status Messages (Level AA) 

### 🔗 What is the consequence of not having this feature?
Screen reader users will not know if their input has been effective. 

### 🚦 How important would you rate the requested feature or improvement?
Important
